examples/ui-demo/src/components/preview/CodePreview.tsx

```diff
@@ -50,9 +52,41 @@ function CodeBlock({ title, code }: { title: string; code: string }) {
   );
 }
 
-const styleCode = `import { withAccountKitUi } from "@alchemy/aa-alchemy/tailwind";
+
+function getTailwindCode(config: Config) {
+  const { ui } = config
+  // TODO: separate primaryColor for light and dark mode
+  return `import { withAccountKitUi, createColorSet } from "@alchemy/aa-alchemy/tailwind";
 import type { Config } from "tailwindcss";
 
-const config: Config = <your tailwind config>;
-export default withAccountKitUi(config);
-`;
+export default withAccountKitUi(<your tailwind config>, {
```

Comment:
something I realized we might need in the alpha getting started doc as we well was the a comment linking to `tailwind` getting started docs, we should add that here as well